src/01-general/users.apib

```diff
@@ -51,6 +51,10 @@ Get a list of all users.
         + filter (object, optional)
             + ids: `cb8da52a-ce89-4bf6-8f7e-8ee6cb85e3b5`,`f8a57a6f-dd1e-41a3-b8d3-428663f1d09e` (array[string], optional)
             + term: `John` (string, optional) - Filters on first name, last name and email
+            + status: `all` (enum[string], optional) - Filters on status
```

Comment:
PP: I would allow an array of status if you want to filter by more than one. We did the same in the `invoice.list` https://github.com/teamleadercrm/api/blob/master/src/05-invoicing/invoices.apib#L16

Reply:
Not sure it makes a lot of sense here though. I understand that for invoices you would have a lot more complex filters. But on users, I am not so sure since we never had the requests to do so.

I'd say that we can always make a change and a BC thing if we actually need it.

Thoughts?

Reply:
I would prefer an array of status, I believe it would be nice if we implement it in the same way in every place. But let's wait to see what @stivni and @jenssegers think about this.

Reply:
I would only implement it where it makes sense...

Reply:
given my comment about `all`, I do think it makes sense here